[2025-01-04 12:14:54,342] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: my_first_dag.Cria_pasta scheduled__2025-01-03T00:00:00+00:00 [queued]>
[2025-01-04 12:14:54,348] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: my_first_dag.Cria_pasta scheduled__2025-01-03T00:00:00+00:00 [queued]>
[2025-01-04 12:14:54,349] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-01-04 12:14:54,349] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2025-01-04 12:14:54,349] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-01-04 12:14:54,365] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): Cria_pasta> on 2025-01-03 00:00:00+00:00
[2025-01-04 12:14:54,367] {standard_task_runner.py:52} INFO - Started process 11145 to run task
[2025-01-04 12:14:54,374] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'my_first_dag', 'Cria_pasta', 'scheduled__2025-01-03T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmp68jpe7rd', '--error-file', '/tmp/tmpd87fr2c5']
[2025-01-04 12:14:54,374] {standard_task_runner.py:80} INFO - Job 13: Subtask Cria_pasta
[2025-01-04 12:14:54,430] {task_command.py:370} INFO - Running <TaskInstance: my_first_dag.Cria_pasta scheduled__2025-01-03T00:00:00+00:00 [running]> on host daniel-virtual-machine
[2025-01-04 12:14:54,508] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_first_dag
AIRFLOW_CTX_TASK_ID=Cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2025-01-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-01-03T00:00:00+00:00
[2025-01-04 12:14:54,509] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-01-04 12:14:54,510] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/daniel/Documents/Alura/pasta"']
[2025-01-04 12:14:54,519] {subprocess.py:85} INFO - Output:
[2025-01-04 12:14:54,526] {subprocess.py:96} INFO - Command exited with return code 0
[2025-01-04 12:14:54,561] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=Cria_pasta, execution_date=20250103T000000, start_date=20250104T151454, end_date=20250104T151454
[2025-01-04 12:14:54,593] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-01-04 12:14:54,619] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
